### PR TITLE
Availability-updates

### PR DESCRIPTION
**Pull request by pawlizio**
Add support for check connectivity updates. Required for https://github.com/pawlizio/my_velux/pull/38

### CODE REVIEW

**Comment by pawlizio** [comment]
@dumpfheimer : I hope you don't mind, as I've closed your PR in my pyvlx and therefore now cherry picked your commits to create this PR in order to avoid that your work goes lost.

File: pyvlx/pyvlx.py [hunk 1]
```diff
@@ -37,20 +37,21 @@ def __init__(
...
-        self.connection.register_frame_received_cb(self.node_updater.process_frame)
         self.nodes = Nodes(self)
+        self.connection.register_frame_received_cb(self.node_updater.process_frame)
         self.scenes = Scenes(self)
         self.version = None
         self.protocol_version = None
         self.klf200 = Klf200Gateway(pyvlx=self)
         self.api_call_semaphore = asyncio.Semaphore(1)  # Limit parallel commands
-        PYVLXLOG.debug("Loadig pyvlx v0.2.21")
+        PYVLXLOG.debug("Loadig pyvlx v0.1.76")
```
Comment (Julius2342) [inline pyvlx/pyvlx.py:54]: Just a side note: we should remove the version from this log message, as this is something which i will always forget to update when bumping a version.

**Review by Julius2342 — APPROVED** [review]
LGTM. May @palfrey also have a look?

**Comment by dumpfheimer** [comment]
> @dumpfheimer : I hope you don't mind, as I've closed your PR in my pyvlx and therefore now cherry picked your commits to create this PR in order to avoid that your work goes lost.

Not at all! Thanks for taking the time!

**Pull request merged**